tests/test_nwm30_grid_loading.py

```diff
@@ -0,0 +1,55 @@
+
+from pathlib import Path
+import filecmp
+
+import pandas as pd
+import pytest
+
+from teehr.loading.nwm30.nwm_grid_data import fetch_and_format_nwm_grids
+from teehr.loading.nwm22.utils_nwm import build_zarr_references
+from teehr.loading.nwm22.const_nwm import (
+    NWM22_UNIT_LOOKUP,
+)
+
+TEST_DIR = Path("tests", "data", "nwm22")
+WEIGHTS_FILEPATH = Path(TEST_DIR, "onehuc10_weights.parquet")
+
+
+@pytest.mark.filterwarnings("ignore::RuntimeWarning")
+def test_grid_loading():
+
+    component_paths = [
```

Comment:
@samlamont should/could thins be two tests?

Reply:
yes good point will do